docs/guide/execute-transaction.md

```diff
@@ -95,21 +95,21 @@ Examples of parameter [`ExecParams`](https://algobuilder.dev/api/algob/modules/r
     payFlags: {}
   }
 ```
-- To learn about more parameters like (account, appArgs, ForeignApps, ForeignAssets etc).Please check [SSCOptionalFlags](https://algobuilder.dev/api/algob/interfaces/runtime.types.sscoptionalflags.html)
+- To learn about more parameters like (account, appArgs, ForeignApps, ForeignAssets etc).Please check [AppOptionalFlags](https://algobuilder.dev/api/algob/interfaces/runtime.types.AppOptionalFlags.html)
 
-### [Opt-In to SSC](https://algobuilder.dev/api/algob/modules/runtime.types.html#optinsscparam)
+### [Opt-In to SSC](https://algobuilder.dev/api/algob/modules/runtime.types.html#OptInAppparam)
 
 ```js
   {
-    type: TransactionType.OptInSSC,
+    type: TransactionType.OptInApp,
     sign: SignType.SecretKey,
     fromAccount: alice,
     appID: appID,
     payFlags: { totalFee: 1000 }
   }
 ```
 
-### [Call SSC](https://algobuilder.dev/api/algob/modules/runtime.types.html#ssccallsparam)
+### [Call SSC](https://algobuilder.dev/api/algob/modules/runtime.types.html#AppCallsParam)
```

Comment:
please update links everywhere. we can verify from building api docs in local

Reply:
ok